js/src.js

```diff
@@ -0,0 +1,377 @@
+/* global L, data */
+let startPolyline = null;
+
+let polygon = null;
+let latLngs;
+let guideline = null;
+let mouseCoord = null;
+
+let polylines = L.layerGroup();
+let markers = L.layerGroup();
+let polygons = L.layerGroup();
+const popupPump =
+    `<b>RSK:</b> 5890162<br>
+<b>ArtikelNr:</b> BPS200<br>
+<b>slang:</b> 32<br>
+<b>invGanga:</b> g 32<br>
+<b>Fas:</b> 1<br>
+<b>Volt:</b> 230<br>
+<b>Motoreffekt:</b> 0.2<br>
+<b>Markström:</b> 1<br>
+<b>varvtal:</b> 2900<br>
+<b>kabeltyp:</b> H05RNF/H07RNF<br>
+<b>kabellängd:</b> 10<br>
+<b>vikt:</b> 5`;
+
+import {
+    map
+} from "./loadLeafletMap.js";
+
+export const object = {
+    /**
+     * Adds a marker to the map.
+     * @param {object} event event.
+     * @returns {void}
+     */
```

Comment:
Comment is not directly above the function it references. Move the assignment of variables on line 36 & 37 to be above the comment or below the function.